File: data/conala-train_code-davinci-002_1666524963/Python/583.py
import requests
from bs4 import BeautifulSoup

url = 'https://www.nytimes.com/'
r = requests.get(url)
r_html = r.text

soup = BeautifulSoup(r_html, "html.parser")

for tag in soup.find_all(class_="story-heading"):
    if tag.a:
        print(tag.a.text.replace("\n", " ").strip())
    else:
        print(tag.contents[0].strip())

File: data/conala-train_code-davinci-002_1666524963/Java/583.java
import requests
from bs4 import BeautifulSoup

url = 'https://www.monster.com/jobs/search/?q=Software-Developer&where=Australia'
page = requests.get(url)
soup = BeautifulSoup(page.content, 'html.parser')
results = soup.find(id='ResultsContainer')
print(results.prettify())